tests/DocumentEmbedderTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Tests;
+
+use PhpLlm\LlmChain\Document\Document;
+use PhpLlm\LlmChain\Document\Vector;
+use PhpLlm\LlmChain\DocumentEmbedder;
+use PhpLlm\LlmChain\EmbeddingsModel;
+use PhpLlm\LlmChain\Store\StoreInterface;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\TestCase;
+
+#[CoversClass(DocumentEmbedder::class)]
+final class DocumentEmbedderTest extends TestCase
+{
+    private EmbeddingsModel $embeddings;
+    private StoreInterface $store;
+    private DocumentEmbedder $embedder;
+
+    protected function setUp(): void
+    {
+        $this->embeddings = $this->createMock(EmbeddingsModel::class);
+        $this->store = $this->createMock(StoreInterface::class);
+        $this->embedder = new DocumentEmbedder($this->embeddings, $this->store);
+    }
+
+    #[Test]
+    public function testEmbedSingleDocument(): void
```

Comment:
i guess gpt was not aware the `test*` prefix is not needed here? :D

Reply:
done